.env

```diff
@@ -13,3 +13,4 @@ REACT_APP_DEFAULT_PATROL_FILTER_FROM_DAYS=1
 # Feature flags
 REACT_APP_LEGACY_RT_ENABLED=false
 REACT_APP_I18N_ENABLED=true
+REACT_APP_EFB_FORM_SCHEMA_SUPPORT_ENABLED=true
```

Comment:
This should be `false` by default. We don't want this to show up in prod envs. Developers must turn it on locally when working on this epic.

Reply:
I left this as true to facilitate QA process, before this PR get merge I will update it